content/applications/productivity/knowledge/articles_creation.rst

```diff
@@ -0,0 +1,242 @@
+
+Articles are organized into four categories:
+
+- :guilabel:`Workspace`: They are available for all internal users in :guilabel:`can read` or
+  :guilabel:`can write` access. Users can invite external users by granting them
+  :guilabel:`can read` access.
+- :guilabel:`Private`: They are only available for the owner.
+- | :guilabel:`Shared`: They are available for the owner who can share the access with other users.
+    This way, the articles are not available for any user except those with access.
+  | Users can invite other internal users by granting them :guilabel:`can read` or :guilabel:`can
+    write` access. They can also invite external users by granting them :guilabel:`can read` access.
+  | For the current user, this section contains all the private articles they shared with someone
+    and all the articles shared with them.
+- :guilabel:`Favorite`: Any identified user can set an article they have access to as a favorite.
+  Doing so is user-specific and has no impact on other users.
+
+.. Future content
+   .. note:: make redirection to "shareability" page to explain how to share
+
+Add articles
+------------
+
+To create articles, click the :guilabel:`Create` button on the right side of the top bar or the
+:guilabel:`+` button next to a category.
+
+An article can have a parent and multiple children. Adding children to an article establishes a
+hierarchy. In a hierarchy, the **root** articles are usually the topic of all the
+articles below them. Adding levels to a hierarchy is not limited. To create a child article,
+click the :guilabel:`+` button next to the parent-to-be article.
+
+.. tip::  Shortcut the creation of same-level articles; press **alt** and **c** simultaneously on
+          your keyboard.
+
+Add and style content
+---------------------
+
+Now, start adding content. To do so, add the first-level heading (h1) at the top of the article,
+press **enter**, and begin writing.
+
+To automatically match your h1 and the article's name, click on :guilabel:`Untitled` on the
+left side of the top bar. Doing so synchronizes the *Untitled article* and the heading.
+
+.. note:: This method only works one time. To modify your article's name, later on, you have to
+   change it manually. To do so, select the article, click on the name on the top bar and proceed to
+   the modification.
+
+Use the Odoo editor
+~~~~~~~~~~~~~~~~~~~
+
+.. image:: articles_creation/text-editor.png
+   :align: center
+
+To open the text editor, select the text you want to format, and the editor appears. Then, click on
+the buttons to change the text style (normal, header 1 --> 6, code, quote), the format (bold,
+italic, underlined, strikethrough, no format), the font and background colors, and the font size.
+You can also create lists and insert (or edit) links.
+
+To format a whole paragraph, type `/` anywhere in the text. Doing so opens the
+:doc:`powerbox <../../../developer/reference/frontend/odoo_editor>`, which allows:
```

Comment:
You can use an absolute path, here, since it goes back to the root of the content, anyway.
```suggestion
:doc:`powerbox </developer/reference/frontend/odoo_editor>`, which allows:
```